region/norfair/east/Bubble Mountain.json

```diff
@@ -336,11 +336,30 @@
       "link": [1, 7],
       "name": "Base",
       "requires": [
+        "Grapple"
+      ],
+      "note": "Kill King Cac by scrolling the camera before Grappling across."
+    },
+    {
+      "link": [1, 7],
+      "name": "Walljump",
+      "requires": [
+        "canWalljump",
```

Comment:
I think I would restructure this into 2 parts:
- WJ + careful jump or precise WJ
- camera manip or dboost

Because even with the cac dead, the jump or WJ is a bit precise